classic_tetris_project/facades/qualifying_types.py

```diff
@@ -238,12 +238,43 @@ def format_score(self):
     def qualifying_data(self):
         return [self.time]
 
+class Clipped7GameAvg(QualifyingType):
+    NAME = "Clipped Seven Game Average"
+    EXTRA_FIELDS = [(f"score{i}", f"Score {i}") for i in range(1, 8)]
+
+    class Form(QualifyingType.Form):
+        score1 = forms.IntegerField(min_value=0, label="Score 1")
+        score2 = forms.IntegerField(min_value=0, label="Score 2")
+        score3 = forms.IntegerField(min_value=0, label="Score 3")
+        score4 = forms.IntegerField(min_value=0, label="Score 4")
+        score5 = forms.IntegerField(min_value=0, label="Score 5")
+        score6 = forms.IntegerField(min_value=0, label="Score 6")
+        score7 = forms.IntegerField(min_value=0, label="Score 7")
```

Comment:
Only thing I wonder is if it's possible for users to not complete 7 games in this format? You could make some of these fields optional, but probably easiest to instruct people to enter 0 as needed

Reply:
Good point, I'll put submitting 0 in the instructions.